azext_connect/custom.py

```diff
@@ -382,10 +383,49 @@ def create_resource(service, resource_group, deployment_id, settings, para_dict)
         spring_cloud_handler(resource_group, deployment_id, settings, para_dict)
 
 
+COSMOSDB_KIND = ['GlobalDocumentDB', 'MongoDB', 'Parse']
```

Comment:
Shall we support this? What't the CLI experience if the input cosmos db is kind of "Parse"?

Reply:
Looks like "Parse" is not currently supported by Azure CosmosDB. But "Parse" is listed in the rest API docs.